Intersect (Core)/Enums/TypewriterTypes.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Intersect.Enums
+{
+    public enum TypewriterTypes
```

Comment:
This should be `TypewriterBehavior` (also `s` is for [Flags], this isn't a flag enum though).

Reply:
Done